site/documentation/content/admin-guide/mongodb-device-registry-config.md

```diff
@@ -27,7 +27,7 @@ The following table provides an overview of the configuration variables and corr
 | `HONO_MONGODB_HOST`<br>`--hono.mongodb.host` | no | `localhost` | The host name or IP address of the MongoDB instance.|
 | `HONO_MONGODB_PORT`<br>`--hono.mongodb.port` | no | `27017` | The port that the MongoDB instance is listening on.|
 | `HONO_MONGODB_PASSWORD`<br>`--hono.mongodb.password` | no | - | The password to use for authenticating to the MongoDB instance.|
-| `HONO_MONGODB_SERVER_SELECTION_TIMEOUT_IN_MS`<br>`--hono.mongodb.serverSelectionTimeoutInMs` | no | `300000` | The time in milliseconds that the mongo driver will wait to select a server for an operation before raising an error.|
+| `HONO_MONGODB_SERVER_SELECTION_TIMEOUT_IN_MS`<br>`--hono.mongodb.serverSelectionTimeoutInMs` | no | `30000` | The time in milliseconds that the mongo driver will wait to select a server for an operation before raising an error.|
```

Comment:
Do you think that 30 seconds is a reasonable default value? The client's timeout is usually somewhere around 200 - 1000ms so I do not think that it makes a lot of sense to wait for 30 seconds in this case or am I mistaken?

Reply:
During the implementation, we left it to the Vert.x default values.I also think 30 seconds is quite high. How about setting it to 1000 ms?

Reply:
Sounds reasonable to me.

Reply:
I have set it to 1000 ms and pushed the changes.